src/services/database/health-record-repository.js

```diff
@@ -168,3 +173,59 @@ export const messageAlreadyReceived = async (messageId) => {
     throw e;
   }
 };
+
+export const findAllSoftDeletedHealthRecords = async () => {
+  const HealthRecord = ModelFactory.getByName(healthRecordModelName);
+
+  return await HealthRecord.findAll({
+    where: {
+      deletedAt: {
+        [Op.ne]: null,
+      },
+    },
+    paranoid: false,
+  })
+    .then((healthRecords) => healthRecords)
```

Comment:
```suggestion
```
I think this line can be removed. We can directly use `.catch` without `.then`. I tried to remove all these .then(healthRecords => healthRecords) in local and integration tests still pass

Reply:
Thank you!